PF/Assignment1/Students.java

```diff
@@ -0,0 +1,81 @@
+package prg2;
+
+import java.util.Arrays;
+
+public class Students {
+	private int top = -1;
+	private int rear = -1;
+	String[] NameList;
```

Comment:
ACTION: Instead of using different array for name and rank make name and rank as property of the student class and use List<Student>.